### ISSUE TITLE
NVDA reads dialog element content even when there is no aria-describedby reference

### ISSUE THREAD

**Issue by Wildebrew** [nvaccess/nvda#17352]
If you open a modal dialog in Chrome and place focus on an element that does not force NVDA into forms mode (e.g. a close button), NVDA reads the entire content of the dialog, even if the dialog element has no aria-describedby reference.

To reproduce. Open a modal dialog that does not have an aria-describedby refernece.
This [dialog from Deque University](https://dequeuniversity.com/library/aria/simple-dialog) will suffice.
Notice how NVDA will read the entire content of the dialog, including labels and buttons, even if the dialog element has no aria-describedby reference.
Jaws only reds the element that receives focus + the role of the dialog.
This is also the expected behavior, as per the ARIA spec (else hy ould authors be able to use aria-describedby to control what dialo content gets read when dialog opens).
This is reproduceable in thee latest version of NVDA 2024.1 (and older versions) with latest versions of both Chrome and Firefox.
Noes:
This is validated with default verbosity settings.

The ARIA Authoring Practices dialog examples are preferred, but most of them put focus in a text input that triggers application mode, where you do not notice this behavior.

I do not consider this a critical or serious issue, but a bit curious, seeing as some modal dialogs do not benefit from being read automatically, e.g. dialogs with a lot of form elements.

**Comment by Adriani90** [comment]
If you disable automatic say all on page load in NVDA browse mode settings, are you then still seeing this issue?
